src/parseBook.py

```diff
@@ -55,7 +53,7 @@ def injectShaderBlocks(_folder, _text):
             gifName, gifExt = os.path.splitext(gifPath)
             pngImage = gifName + ".png"
             convertCommand = "convert " + gifPath + " " + pngImage
-            print convertCommand
+            print(convertCommand
```

Comment:
Also need closing parentheses here.. sorry if this is wrong!

Reply:
Indeed, I'd forgotten these. Fixing.